cspell.yaml

```diff
@@ -73,12 +75,14 @@ words:
   - inotify
   - iowait
   - ipapi
+  - itertools
   - iwctl
   - kbdd
   - kbddbus
   - kibi
   - kmon
   - libc
+  - libdbus
```

Comment:
Why was "libdbus" added? I don't think it is ever mentioned anywhere else.

Reply:
It's left over from when I was building pipewire from src. I'll remove it now.